slimeworldmanager-plugin/src/main/java/com/grinderwolf/swm/plugin/commands/sub/WorldListCmd.java

```diff
@@ -90,4 +91,9 @@ public boolean onCommand(CommandSender sender, String[] args) {
 
         return true;
     }
+
+    @Override
+    public List<String> onTabComplete(CommandSender sender, String[] args) {
+        return new LinkedList<>();
```

Comment:
I'd rather return `Collections.emptyList()` instead of a new list every time there are no available completions. Creating a list every time is a waste of memory. Same goes with all the other subcommands.